packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/plugins/ui/components.tsx

```diff
@@ -227,16 +232,17 @@ export const components = () => {
         {...props}
       />
     ),
-    [MARK_CODE]: ({ editor, leaf, text, attributes, className, ...props }) => (
-      <code
-        className={classNames('bg-gray-100 p-1 rounded-sm', className)}
-        {...attributes}
-        {...props}
-      />
-    ),
-    [MARK_ITALIC]: ({ editor, leaf, text, ...props }) => (
-      <em {...props.attributes} {...props} />
-    ),
+    [MARK_CODE]: CodeLeaf,
+
+    // [MARK_ITALIC]: ({ editor, leaf, text, ...props }) => (
+    //   <em {...props.attributes} {...props} />
+    // ),
+    // [ELEMENT_UL]: withProps(ListElement, { variant: "ul" }),
+    // [ELEMENT_OL]: withProps(ListElement, { variant: "ol" }),
+    // [ELEMENT_LI]: withProps(PlateElement, { as: "li" }),
```

Comment:
**question**: Is this commented-out code worth retaining?

Reply:
Removed